include/boost/http_proto/serializer.hpp

```diff
@@ -293,48 +338,147 @@ class BOOST_SYMBOL_VISIBLE
     bool is_done_;
     bool is_chunked_;
     bool is_expect_continue_;
+    bool is_compressed_ = false;
 };
 
 //------------------------------------------------
 
+/**
+    A proxy type used to pass bounded input to the
+    associated serializer.
+*/
 struct serializer::stream
 {
-    /** Constructor.
+    /** Default constructor.
+
+        Creates a stream without an associated serializer
+        object.
```

Comment:
It would be better to explain the consequences instead of that it merely doesn't have an associated serializer. For example "The only valid operations on default constructed streams are assignment and destruction."